src/components/ChatLog.jsx

```diff
@@ -0,0 +1,46 @@
+import ChatEntry from './ChatEntry';
+import PropTypes from 'prop-types';
+
+
+const ChatLog = (props) => {
+  const chatentrycomponents = props.entries.map((entries, index) => {
+    return (
+      <ul key={index}>
+        <ChatEntry
+          id={entries.id}
+          sender={entries.sender}
+          body={entries.body}
+          timeStamp={entries.timeStamp}
+          liked={entries.liked}
+          onLiked={props.onLiked}
+          localSender={props.localSender}
+        ></ChatEntry>
+      </ul>
+    );
+  });
+
+  return (
+    <section>
+      <ul className="chat-log">
+        {chatentrycomponents}
+      </ul>
+    </section>
+  );
+};
+
+ChatLog.propTypes = {
+  entries: PropTypes.arrayOf(
+    PropTypes.shape({
+      id: PropTypes.number.isRequired,
+      sender: PropTypes.string.isRequired,
+      body: PropTypes.string,
+      timeStamp: PropTypes.string.isRequired,
+      liked: PropTypes.bool.isRequired,
+    })
+  ).isRequired,
+
+  onLiked: PropTypes.func.isRequired,
+  localSender: PropTypes.string.isRequired,
```

Comment:
⭐️

Reply:
I see that the tests are saying that some of the props are undefined but that isn't the case. I'm not entirely sure why.